setup.py

```diff
@@ -34,12 +34,12 @@ def read_file(fname):
 
 
 requirements = [
-    "Django>=1.8,<2.0",
+    "Django>=1.8,<2.1",
     "bleach>=1.5,<2",
     "Pillow",
-    "django-nyt>=1.0b1",
+    "django-nyt>1.0b6",
     "six",
-    "django-mptt>=0.8.6,<0.9",
+    "django-mptt>=0.8.6",
```

Comment:
Removing the upper bounds for django-mptt isn't a good idea. This has historically caused brokenness for deployments.